core/dvector.h

```diff
@@ -630,13 +595,12 @@ template <class T>
 void PoolVector<T>::invert() {
 	T temp;
 	Write w = write();
-	int s = size();
-	int half_s = s / 2;
+	int len = size();
 
-	for (int i = 0; i < half_s; i++) {
+	for (int i = 0; i < len / 2; i++) {
```

Comment:
This change is less optimized than before, here len / 2 has to be calculated on every loop.

Reply:
Good spot, I didn't consider the expression being evaluated each loop.
I've updated it now.